### PR TITLE
Use readline for reading from stdin

### LINKED ISSUE #86: Error: EAGAIN: resource temporarily unavailable, read
```
$ ebi --version
1.2.0
```

```
$ http 'https://tako.in.ft.com/tako/repositories?topic=serverless' "Authorization:Bearer $(heroku config:get --app ft-next-tako BEARER_TOKEN)" | jq -r '.repositories[] | "\(.owner)/\(.name)"' | ebi --token $VAULT_AUTH_GITHUB_TOKEN contents app.json
/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/node_modules/yargs/yargs.js:1163
      else throw err
           ^

Error: EAGAIN: resource temporarily unavailable, read
    at Object.fs.readSync (fs.js:675:18)
    at tryReadSync (fs.js:540:20)
    at Object.fs.readFileSync (fs.js:583:19)
    at getRepositories (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/lib/get-repositories.js:16:20)
    at Object.exports.handler (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/src/commands/contents.js:50:35)
    at Object.runCommand (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/node_modules/yargs/lib/command.js:242:26)
    at Object.parseArgs [as _parseArgs] (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/node_modules/yargs/yargs.js:1078:30)
    at Object.get [as argv] (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/node_modules/yargs/yargs.js:1012:21)
    at Object.<anonymous> (/Users/samuel.parkinson/.nvm/versions/node/v8.15.1/lib/node_modules/ebi/bin/ebi.js:13:32)
    at Module._compile (module.js:653:30)
```

### PR DESCRIPTION
**Pull request by taktran**
Replacing readFileSync to prevent `EAGAIN` errors.

Fixes #86

TODO:

* [x] Fix `getRepositories` test
* [x] Add test for `readStardardInputLines`

### CODE REVIEW

**Comment by taktran** [comment]
Tests are still broken, but `readline` seems to work on the command line

**Comment by taktran** [comment]
For future reference. May or may not be useful: https://github.com/caitp/node-mock-stdin

**Comment by taktran** [comment]
Tests working now, so ready for review.

File: lib/read-standard-input-lines.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const readline = require('readline');
+
+function readStardardInputLines() {
+	const stdinRepos = readline.createInterface({
```
Comment (sjparkinson) [inline lib/read-standard-input-lines.js]:
```suggestion
	const readlineInterface = readline.createInterface({
```

Comment (sjparkinson) [reply lib/read-standard-input-lines.js]: Given this is a generic function.

**Pull request merged**